pages/1.13/security/oss/managing-users/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle:  Managing users
+title: Managing users
+menuWeight: 0
+excerpt: Managing users
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS can manage two types of users:
+
+* **External**: DC/OS never receives or stores the passwords of external users. Instead, it delegates the verification of the user’s credentials to the Auth0 identity provider instance managed by Mesosphere. Only the user ID of one of the three available login providers Google, GitHub or Microsoft is stored inside DC/OS.
```

Comment:
Maybe move this around a bit so the first sentence clearly states what an external user is.

"Users who login with their Google, GitHub, or Microsoft credentials. Verification of the user's credentials is delegated to one of these providers, through Mesosphere's Auth0 instance. None of Auth0, Mesosphere or the DC/OS cluster receive or store the passwords of external users."